src/networking/systems.rs

```diff
@@ -0,0 +1,240 @@
+use std::{collections::HashMap};
+
+use bevy::prelude::{Commands, Entity, EventReader, Events, ResMut};
+use quinn::{crypto::rustls::TlsSession, generic::{RecvStream, SendStream}};
+use tokio::{stream::StreamExt, sync::mpsc::UnboundedReceiver, sync::mpsc::UnboundedSender, sync::mpsc::unbounded_channel};
+use tracing::{error, info};
+
+use super::{components::Connection, events::ReceiveEvent, events::SendEvent, id::ConnectionId, id::StreamId, packets::Packet, streams::BoundedPlanetRecvStream, streams::BoundedPlanetSendStream};
```

Comment:
Should be unrolled again.

Reply:
The exact verbosity is up to you, but something link this:
```suggestion
use crate::networking::{
    events::{
        ReceiveEvent,
        SendEvent
    },
    systems::{
        NetworkConnections,
        SessionEventListenerState,
        handle_connection,
        receive_net_events,
        send_net_events
    }
};
```